Library/Homebrew/cmd/test-bot.rb

```diff
@@ -91,13 +92,31 @@ def failed?
 
     def puts_command
       cmd = @command.join(" ")
-      print "#{Tty.blue}==>#{Tty.white} #{cmd}#{Tty.reset}"
+      cmd_line = "#{Tty.blue}==>#{Tty.white} #{cmd}#{Tty.reset}"
+      if ENV["TRAVIS"]
+        @travis_timer_id = rand(2**32).to_s(16)
+        puts "travis_fold:start:#{@command.join(".")}"
+        puts "travis_time:start:#{@travis_timer_id}"
+        puts cmd_line
+        return
+      end
```

Comment:
I usually use `early return` in nested or complex conditions. To me, using it in simple binary conditions makes the code inbalanace or asymmetric. Anyway, your call.